Pages/Authentication/login.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Login</title>
+    <link href="/style.css" rel="stylesheet" type="text/css" />
+    <link href="https://spazeir-ui.netlify.app/Components/Input/input.css" rel="stylesheet" type="text/css" />
+    <link href="https://spazeir-ui.netlify.app/Components/Button/button.css" rel="stylesheet" type="text/css" />
+    <link href="login.css" type="text/css" rel="stylesheet" />
+</head>
+
+<body>
+    <header class="hero-header">
+        <p class="hero-brand-name">Spazeir Mart</p>
+        <a href="/index.html" class="hero-category-name">
+            <p>Grocery</p>
+        </a>
+        <input class="input-textbox" type="text" placeholder="Search here..." name="text" />
+        <button class="button primary-yellow">Login</button>
+        <a href="/Pages/Cart management/cartmanagement.html"><span class="fab fa-solid fa-cart-shopping"></span></a>
+        <a href="/Pages/Wishlist/wishlist.html"><span class="fab fa-solid fa-heart"></span></a>
+    </header>
+
+    <main class="main-container">
+        <div class="login-container">
+            <h3>
+                <center>Login</center>
```

Comment:
Center tag is deprected in HTML 5. Read more [here](https://www.w3schools.com/tags/tag_center.asp)

Reply:
Thank you for sharing this